content/docs/integrations/kubernetes/index.md

```diff
@@ -22,7 +22,60 @@ Roadie supports the following connection setups:
 
 All three of these can use a Broker service that runs in your infrastructure and means that you do not have to expose your cluster via an AWS cross account Role, Google OAUTH App token or Service Account Token.
 
-# Via Broker
+# Kubernetes Annotations
+
+The k8s plugin uses two alternate ways of discovering resources in addition to a namespace annotation. You can find the Backstage docs on this [here](https://backstage.io/docs/features/kubernetes/configuration/#surfacing-your-kubernetes-components-as-part-of-an-entity).
+
+## 1. backstage.io/kubernetes-id
+You can add a label with the key `backstage.io/kubernetes-id` to all of your k8s resources relating to an entity and then using the same key as an annotation on your entity.
+
+Additionally, you can lookup resources by namespace using the `backstage.io/kubernetes-namespace` annotation. 
```

Comment:
I'm assuming the default is "default"

Reply:
Oh well I just tested that and I think I'm wrong there.

Reply:
No that was just a misconfig